core/cog-shell.c

```diff
@@ -127,7 +127,7 @@ cog_shell_class_init (CogShellClass *klass)
 }
 
 static void
-cog_shell_init (CogShell *shell)
+cog_shell_init (G_GNUC_UNUSED CogShell *shell)
```

Comment:
Now that I see this it dawned on me that we do not place the `G_GNUC_UNUSED` annotations consistently in all the code base, sometimes they are before the type, sometimes after the variable name… most of them are _after_ the variable names, so I say let's go with that from now on. Could you move this one so it reads `CogShell *shell G_GNUC_UNUSED`?